python/sklearnserver/custom_transformer/custom_transformer.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 The KServe Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pandas as pd
+from sklearn.base import TransformerMixin
+
+
+class DictToDFTransformer(TransformerMixin):
```

Comment:
This file is only used for sklearn pipeline example right?

Reply:
yes @yuzisun . It is used only for sklearnserver model test.